pkg/memory/memory.go

```diff
@@ -16,7 +16,7 @@ type Gomem struct {
 	IsWow64       bool
 }
 
-func NewGomem[T uint32 | string](processId T, desiredAccess *uint32) (*Gomem, error) {
```

Comment:
I will accept that PR, but remove this change first

Reply:
OK. I'll edit it later.